fourmolu.yaml

```diff
@@ -0,0 +1,10 @@
+indentation: 2
+function-arrows: leading
```

Comment:
I don't recommend this - with any special arrows, the arrow actually refers to the thing on the left, so e.g. with linear types:
```haskell
f
  :: Int
  %1 -> Bool
```
The special arrow is referring to the Int on the previous line, so to parse this in your mind you need to keep the context from the previous line.

```haskell
f :: 
  Int %1 -> 
  Bool
```

Also personally I don't care to have to mentally skip over the leading arrow - it's just unnecessary noise in most cases and I want to get to the type faster

Reply:
Alrighr, I'll reconsider and try out other variants